src/app/Shared/Services/Api.service.tsx

```diff
@@ -1134,6 +1136,33 @@ export class ApiService {
     );
   }
 
+  downloadDashboardLayout(layout: DashboardLayout): void {
+    const serializedLayout = this.getSerializedDashboardLayout(layout);
+    const filename = `${layout.name}${nanoid(5)}.json`;
```

Comment:
This filename with the random ID ends up being a little weird looking. If there is no random ID, I think the browser should (hopefully) handle any on-disk naming collisions for us?

Also I'd suggest putting something like `cryostat-dashboard-` as a prefix so that it's more obvious to the user what this `.json` file is later when they see it in their downloads directory.